spec/javascript/packages/document-capture/components/review-issues-step-spec.jsx

```diff
@@ -198,6 +199,47 @@ describe('document-capture/components/review-issues-step', () => {
     );
   });
 
+  it('does not render sp help troubleshooting option for errored review', () => {
+    const { queryByRole } = render(
+      <InPersonContext.Provider value={{ inPersonURL: null }}>
+        <ServiceProviderContextProvider
+          value={{
+            name: 'Example App',
+            failureToProofURL: 'https://example.com/?step=document_capture',
+          }}
+        >
+          <ReviewIssuesStep {...DEFAULT_PROPS} />
+        </ServiceProviderContextProvider>
+      </InPersonContext.Provider>,
+    );
+
+    expect(
+      queryByRole('link', { name: 'idv.troubleshooting.options.get_help_at_sp links.new_window' }),
+    ).to.not.exist();
+  });
+
+  it('does render sp help troubleshooting option for errored review if in person url present', () => {
```

Comment:
Very minor and definitely not warranting a follow-up pull request, but: the "if in person url present" reads to me as a "context", so I might have imagined a hierarchy like...

```ts
it('does not render sp help troubleshooting option for errored review', () => {});

context('if in person url present', () => {
  it('does render sp help troubleshooting option for errored review', () => {});
});
```